doc/basics/compilation.md

```diff
@@ -70,7 +70,7 @@ RADIUM_GENERATE_LIB_CORE:BOOL=ON
 // Include Radium::Engine in CMake project
 RADIUM_GENERATE_LIB_ENGINE:BOOL=ON
 --
-// Include Radium::GuiBase in CMake project
+// Include Radium::Gui in CMake project
```

Comment:
Shouldn’t we rename the variable as well, for consistency ?

Reply:
Actually it's the case in CMakeLists.txt, need update here

Reply:
(it was modified in another PR, I mess up in the pushes)